src/httpClient.ts

```diff
@@ -101,21 +101,72 @@ export default class httpClient {
         };
     }
 
-    async postMetaAnalysis(records: AirtableRecord[], aggregation_variable: AggregationFactor, meta_analysis_technique: string = 'fixed', meta_analysis_transformation: string = 'double_arcsin_precise'){
+    async getEstimateGrades(records: AirtableRecord[]) {
+        const formattedRecords = records.map(o => {
+            return {
+                country: o.country,
+                estimate_grade: o.estimate_grade,
+                serum_pos_prevalence: o.seroprevalence,
+                denominator: o.denominator
+            }
+        })
+        const req_body = {
+            records: formattedRecords
+        }
+        const response = await this.httpPost('/airtable_scraper/country_seroprev_summary', req_body);
+        if (!response) {
+            return [];
+        }
+        const airtable_records = response.map((item: Record<string, any>) => {
+            // Convert response to AlternateAggregatedRecord type            
+            const itemName = Object.keys(item)[0];
+            const record = item[itemName]
+            const estimateSummary = record.seroprevalence_estimate_summary;
+            return {
+                numberOfStudies: record.n_estimates,
```

Comment:
why don't we just ask @simonarocco to format the response in this way instead of having to do this transformation??

Reply:
I know I've done this in the past but I don't think this is a good pattern, we can learn from my past mistakes